CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if(POLICY CMP0057)
     cmake_policy(SET CMP0057 NEW) # support IN_LIST
 endif()
 
-
+add_compile_options(-Wno-overloaded-virtual)
```

Comment:
IIRC, you are going to **include** this project. could you only apply this option to certain targets built from aws-sdk-cpp instead of to the whole Ceph project?

Reply:
done